src/Taxonomies/Term.php

```diff
@@ -36,9 +38,14 @@ public function id()
 
     public function slug($slug = null)
     {
-        return $this->fluentlyGetOrSet('slug')->setter(function ($slug) {
-            return Str::slug($slug);
-        })->args(func_get_args());
+        return $this
+            ->fluentlyGetOrSet('slug')
+            ->setter(function ($slug) {
+                $this->syncOriginal();
```

Comment:
We shouldn't be syncing the original here.

Reply:
🤔